src/styles/cards-block/card.scss

```diff
@@ -0,0 +1,137 @@
+.conatiner {
```

Comment:
There's a typo in the class name. It seems like it should be '.container' instead of '.conatiner'.